dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -641,6 +663,7 @@ public void setOutputFileFormats(final SortedSet<FileFormat> outputFileFormats)
      * @return versions
      */
     @JsonProperty
+    @ArraySchema(uniqueItems = true, schema = @Schema(description = "foo", implementation = WorkflowVersion.class))
```

Comment:
Can't Tag.class be an implementation as well?

Description needs to be filled out.